docs/insets.md

```diff
@@ -2,6 +2,100 @@
 
 [![Maven Central](https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-insets)](https://search.maven.org/search?q=g:com.google.accompanist)
 
+!!! warning
```

Comment:
```suggestion
> :warning:
```

Reply:
Just thought this might look cleaner

Reply:
I'm not entirely familiar with the doc syntax, but plan was to getting a warning block similar to the permissions one:
https://github.com/google/accompanist/blob/d9ef702b78c0eca859fa9a1697fc13a6d0526aee/docs/permissions.md?plain=1#L7

Reply:
Neither am I to be honest, I was basing that on the markdown preview but I think you are right